src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-snapshot-form/rbd-snapshot-form-modal.component.ts

```diff
@@ -53,7 +54,7 @@ export class RbdSnapshotFormModalComponent extends BaseModal implements OnInit {
   createForm() {
     this.snapshotForm = new CdFormGroup({
       snapshotName: new UntypedFormControl('', {
-        validators: [Validators.required]
+        validators: [Validators.required, CdValidators.validateNoDotOrSlash()]
```

Comment:
We can use [angular pattern validator](https://v17.angular.io/api/forms/Validators#pattern) and pass a regex to it instead of creating a custom validator isn't it?